UefiTestingPkg/FunctionalSystemTests/MemoryProtectionTest/App/AArch64/AArch64Functions.c

```diff
@@ -0,0 +1,46 @@
+/** @file
+
+Copyright (C) Microsoft Corporation. All rights reserved.
+SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#include <Uefi.h>
+
+#include <Library/DebugLib.h>
+#include <Library/UnitTestLib.h>
+
+/**
+  Registers the interrupt handler which will be invoked when a page fault occurs.
+
+  @retval     EFI_SUCCESS         Interrupt handler successfully installed
+  @retval     EFI_UNSUPPORTED     Installing the interrupt handler is not supported.
+  @retval     others              Error occurred during installation.
+**/
+EFI_STATUS
+EFIAPI
+RegisterMemoryProtectionTestAppInterruptHandler (
+  VOID
+  )
+{
+  // TODO: Add support for reset test method on Arm
+  return EFI_UNSUPPORTED;

```

Comment:
Shouldn't ARM support similar exception handling as the x64 routine?

Reply:
Yes, but I need to refresh which synchronous exceptions will occur on translation fault and it will take a quite a while to actually test that solution so I have it on the back burner for now.